test/e2e/resources/components/roIntrinsics.brs

```diff
@@ -17,5 +18,7 @@ sub main()
     print "Double to string "doubleObject.toStr()
     print "Float to string "floatObject.toStr()
     print "Integer to string "integerObject.toStr()
+    print "LongInteger object "longIntegerObject
```

Comment:
this test look pretty similar to the one below, can you check that the type is correct instead here?

Reply:
done